terraform/environment/region/lambda.tf

```diff
@@ -25,7 +25,7 @@ module "lambda" {
     DDB_TABLE_NAME_CHANGES  = var.dynamodb_name_changes
     EVENT_BUS_NAME          = var.event_bus.name
     S3_BUCKET_NAME_ORIGINAL = var.lpa_store_static_bucket.bucket
-    JWT_SECRET_KEY_ID       = "${var.account_name}/jwt-key"
+    JWT_SECRET_KEY_ID       = data.aws_secretsmanager_secret.jwt_secret_key.arn
```

Comment:
This (and places that reference it) should probably be `JWT_SECRET_KEY_ARN` now

Reply:
done. your comment also made me go and check what the last part of the ARN is (`arn:aws:secretsmanager:Region:AccountId:secret:a_specific_secret_name-a1b2c3`) because I thought at one point that it may refer to the version of the secret, but it isn't.

Thanks